coffea/nanoaod/methods/jets.py

```diff
@@ -4,6 +4,7 @@
 
 class FatJet(LorentzVector):
     '''NanoAOD large radius jet object'''
+    _enable_genjet = False
     subjetmap = {'FatJet': 'SubJet'}  # V6 has 'GenJetAK8': 'SubGenJetAK8', maybe better to put in generator.py
```

Comment:
Ah I think here is why I had set `GenJetAK8` to map to FatJet class: since you can link the subjets. But as the comment says, this probably makes more sense as a specialized class. For now LorentzVector is probably good enough.

Reply:
Hmm yeah wanting that makes as much sense as wanting FatJet to GenJet match, tbh I am not quite sure why this was a problem.